### ISSUE TITLE
Default tab fonts don't meet WCAG recommended contrast ratios

### ISSUE THREAD

**Issue by RyanJenkins** [qutebrowser/qutebrowser#3449]
The contrast ratio between the default tab text color (white) and background color (gray and darkgray) fail WCAG 2.0 contrast ratio recommendations. Just anecdotally, even without any visual impairment the white-on-darkgray text can be difficult to read. I think it would be reasonable to default to something that's widely accessible and allow those who want the lower contrast ratio to alter settings rather than the reverse.

Using #3D3D3D and #595959 for tab background colors WCAG formula and looks reasonable, screenshot of that color scheme attached.

![screen shot 2017-12-30 at 1 40 01 pm](https://user-images.githubusercontent.com/2343714/34457543-7c9fe14a-ed68-11e7-87ff-576b0356b41d.png)

**Comment by nemanjan00** [comment]
I actually even think this one looks cooler...

**Comment by The-Compiler** [comment]
Seems fine to me! I'm against changing the colors in a "backwards-incompatible" way (there was someone proposing a lighter rather than darker color for the selected tab), but this seems like a small enough (and warranted) change.

However, I find it kind of hard to tell odd/even tabs apart like this. It's certainly less important than readable text, though.

**Comment by The-Compiler** [comment]
I've tried this for a while, and found it very hard to tell what tab is active with those changed colors:

![image](https://user-images.githubusercontent.com/625793/36963969-81c1035a-2055-11e8-894b-4881f4200c49.png)

Still open for suggestions on how to improve this without making it hard to see what tab is active, and ideally without confusing current qutebrowser users with big color changes.

**Comment by mschilli87** [comment]
I didn't try, but how about inverting the colors for the active tab (grey on white)?
